mediawiki/LocalSettings.php

```diff
@@ -18,8 +18,8 @@
 ## Uncomment this to disable output compression
 # $wgDisableOutputCompression = true;
 
-$wgSitename = "Dicionário de Favelas Marielle Franco";
-$wgMetaNamespace = "Wikifavelas";
+$wgSitename = "Mapa movimentos";
+$wgMetaNamespace = "MapaMovSaude";
```

Comment:
Por conta deste arquivo ser utilizado por 2 projetos, creio que para estas variáveis convém utilizarmos environments vindas do .env do projeto raiz que é o https://github.com/FiocruzLivre/mediawiki-development.

Acho que pode ser uma boa colocar no `.env`  2 environments substituindo o padrão camel case para upper case com snake case que é um padrão mais comum para nomes de environments:

```env
WG_SITENAME="Mapa movimentos"
WG_META_NAMESPACE="MapaMovSaude"
```

E nestas duas variáveis utilizar:


```suggestion
$wgSitename = getenv('WG_SITENAME');
$wgMetaNamespace = getenv('WG_META_NAMESPACE');
```